index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div id="main">
+        <section id="navigation">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a href="index.html">Youtube</a></li>
+                </ul>
+            </nav>
+        </section>
```

Comment:
Kindly use <header> instead of <section>. See the when to use header [here](https://www.geeksforgeeks.org/html5-header-tag/) and when to use section [here](https://blog.hubspot.com/website/section-in-html)